docs/docs/build/examples/wbtc-gas-fee-erc-2771/_category_.yml

```diff
@@ -0,0 +1,3 @@
+position: 4
```

Comment:
Let's merge both AA and MT into the how to section under a combined title called "Using bridged BTC as Transaction Fees".

The first paragraph should then be a comparison of AA and MT with their trade-offs (can be taken from the knowledge sharing presentation)